src/main/java/core/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.strategy;
+
+import core.model.FruitTransaction;
+
+public class OperationStrategyImpl implements OperationStrategy {
+
+    @Override
+    public OperationHandler getOperationHandler(FruitTransaction.Operation operation) {
+        switch (operation) {
+            case BALANCE:
+                return new BalanceOperationHandler();
+            case SUPPLY:
+                return new SupplyOperationHandler();
+            case PURCHASE:
+                return new PurchaseOperationHandler();
+            case RETURN:
+                return new ReturnOperationHandler();
```

Comment:
this should be a map
when creating OperationStrategyImpl instance pass that map with a constructor